cg/services/illumina_services/illumina_post_processing_service/illumina_post_processing_service.py

```diff
@@ -27,36 +30,51 @@ def __init__(self, status_db: Store, housekeeper_api: HousekeeperAPI, dry_run: b
 
     @staticmethod
     def store_illumina_flow_cell(
-        flow_cell: FlowCellDirectoryData,
+        flow_cell_dir_data: FlowCellDirectoryData,
```

Comment:
I think the other way around would make more sense. however since we will have multiple different flow cell types soon we should revisit that I think. Here I will keep the more explicit way to refer to flow cell dir data . When writing the code it was confusing enough at times...